### PR TITLE
[sensors_plus] Migrate sensors code to sensors_plus

### PR DESCRIPTION
**Pull request by bbrto21**
This PR includes:
* [sensors_plus] Copy from sensors 
* [sensors_plus] Migrate to plus plugin
* [sensors_plus] Update README.md
* [sensors_plus] Copy test_driver from sensors_plus

### CODE REVIEW

**Comment by bbrto21** [comment]
#116

File: packages/sensors_plus/LICENSE [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2017 The Chromium Authors. All rights reserved.
```
Comment (swift-kim) [inline packages/sensors_plus/LICENSE:2]: Here are my review comments for battery_plus_tizen:

- https://github.com/flutter-tizen/plugins/pull/120#discussion_r644451282
- https://github.com/flutter-tizen/plugins/pull/120#discussion_r644451413

File: packages/sensors_plus/example/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  flutter:
+    sdk: flutter
+  sensors_plus: ^1.0.0
+  sensors_plus_tizen:
+    path: ../
+
+dev_dependencies:
+  flutter_driver:
+    sdk: flutter
+  integration_test: ^1.0.2
```
Comment (swift-kim) [inline packages/sensors_plus/example/pubspec.yaml:15]: Didn't this work?

```suggestion
  integration_test:
    sdk: flutter
```

File: README.md [hunk 3]
```diff
@@ -47,6 +48,7 @@ The _"non-endorsed"_ status means that the plugin is not endorsed by the origina
 | [**path_provider_tizen**](packages/path_provider) | ⚠️ | ⚠️ | ⚠️ | ⚠️ | No external storage |
 | [**permission_handler_tizen**](packages/permission_handler) | ✔️ | ✔️ | ❌ | ❌ | Not applicable for TV |
 | [**sensors_tizen**](packages/sensors) | ✔️ | ✔️ | ❌ | ❌ | No hardware |
+| [**sensors_plus_tizen**](packages/sensors_plus) | ✔️ | ✔️ | ❌ | ❌ | No hardware |
```
[the remark this answers is not in the document]
Comment (bbrto21) [reply README.md:51]: thanks!

File: packages/sensors_plus/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# sensors_plus_tizen
+
+The Tizen implementation of ['sensors_plus'](https://github.com/fluttercommunity/plus_plugins/tree/main/packages/sensors_plus).
```
[the remark this answers is not in the document]
Comment (bbrto21) [reply packages/sensors_plus/README.md:3]: I fixed it!

File: packages/sensors_plus/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Usage
+
+This package is not an _endorsed_ implementation of 'sensors_plus'. Therefore, you have to include `sensors_plus_tizen` alongside 'sensors_plus' as dependencies in your `pubspec.yaml` file.
+
+```yaml
+dependencies:
+  sensors_plus: ^1.0.0
+  sensors_plus_tizen: ^2.0.0
```
[the remark this answers is not in the document]
Comment (bbrto21) [reply packages/sensors_plus/README.md:12]: done!

File: packages/sensors_plus/LICENSE (hunk 1 above)
Comment (bbrto21) [reply packages/sensors_plus/LICENSE:2]: I missed your review! thank you~

File: packages/sensors_plus/example/pubspec.yaml (hunk 2 above)
Comment (bbrto21) [reply packages/sensors_plus/example/pubspec.yaml]: After listening to your explanation offline, I've got to know that what you suggested is a more modern way. thanks!

File: README.md [hunk 6]
```diff
@@ -23,6 +23,7 @@ The _"non-endorsed"_ status means that the plugin is not endorsed by the origina
 | [**path_provider_tizen**](packages/path_provider) | [path_provider](https://github.com/flutter/plugins/tree/master/packages/path_provider) (1st-party) | [![pub package](https://img.shields.io/pub/v/path_provider_tizen.svg)](https://pub.dev/packages/path_provider_tizen) | No |
 | [**permission_handler_tizen**](packages/permission_handler) | [permission_handler](https://github.com/Baseflow/flutter-permission-handler) (3rd-party) | [![pub package](https://img.shields.io/pub/v/permission_handler_tizen.svg)](https://pub.dev/packages/permission_handler_tizen) | No |
 | [**sensors_tizen**](packages/sensors) | [sensors](https://github.com/flutter/plugins/tree/master/packages/sensors) (1st-party) | [![pub package](https://img.shields.io/pub/v/sensors_tizen.svg)](https://pub.dev/packages/sensors_tizen) | No |
+| [**sensors_plus_tizen**](packages/sensors_plus) | [sensors_plus](https://github.com/fluttercommunity/plus_plugins/tree/main/packages/sensors_plus) (3st-party) | [![pub package](https://img.shields.io/pub/v/sensors_plus_tizen.svg)](https://pub.dev/packages/sensors_plus_tizen) | No |
```
Comment (HakkyuKim) [inline README.md:26]: Just one suggestion!
```suggestion
| [**sensors_plus_tizen**](packages/sensors_plus) | [sensors_plus](https://github.com/fluttercommunity/plus_plugins/tree/main/packages/sensors_plus) (3rd-party) | [![pub package](https://img.shields.io/pub/v/sensors_plus_tizen.svg)](https://pub.dev/packages/sensors_plus_tizen) | No |
```

**Review by bwikbs — APPROVED** [review]
Thanks!! :+1:

**Review by swift-kim — APPROVED** [review]

**Review by HakkyuKim — APPROVED** [review]

**Pull request merged**